back/engines/free/frontend/app/services/frontend/url_service.rb

```diff
@@ -24,8 +24,14 @@ def model_to_path(model_instance)
       when User
         subroute = 'profile'
         slug = model_instance.slug
-      when Comment, InternalComment, OfficialFeedback ### comments and official feedbacks do not have a path yet, we return the post path for now
+      when Comment, OfficialFeedback # Comments and official feedbacks do not have a path yet, we return the post path for now
         return model_to_path(model_instance.post)
+      when InternalComment # Internal comments are only implemented in the Back Office / Admin UI
+        if model_instance.post_type == 'Idea'
+          return "/admin/projects/#{model_instance.post.project_id}/ideas/#{model_instance.post.id}##{model_instance.id}"
```

Comment:
Could you please check with @EdwinKato to find where this route is defined on the FE, and then leave a comment there like "If you change this route, you'll need to update it in `back/engines/free/frontend/app/services/frontend/url_service.rb:31`?